go/libraries/doltcore/remotestorage/chunk_store.go

```diff
@@ -91,6 +91,7 @@ func NewDoltChunkStoreFromPath(ctx context.Context, nbf *types.NomsBinFormat, pa
 		return nil, ErrInvalidDoltSpecPath
 	}
 
+	// todo:
```

Comment:
Dangling...

Reply:
this was just to annotate the line below it without changing the blame. might have overthought that